.travis.yml

```diff
@@ -80,7 +99,14 @@ before_install:
   - mkdir -p $HOME/Downloads $HOME/Deps
 
 install:
-  - travis_wait 30 ./testing/dependencies/install_dependencies.sh
+  - ./testing/dependencies/travis/install.sh
+  - ./testing/dependencies/travis/cmake.sh
+  - ./testing/dependencies/travis/ninja.sh
+  - if [[ "$ANACONDA_TESTS_ONLY" == 1 ]]; then
+      ./testing/dependencies/travis/anaconda.sh;
+    else
+      ./testing/dependencies/travis/eigen.sh;
```

Comment:
Should this always be installed?

Reply:
Not sure I understand the question but I guess it will answer itself if I remove the if-check.

Reply:
Yeah, it answered itself :smile: